README.md

```diff
@@ -23,16 +23,24 @@ to JS by the [awscrt](https://github.com/awslabs/aws-crt-nodejs) package.
 
 
 ## Installation
-### Minimum Requirements
-*   Node 10.x+
+### Check for minimum Requirements
+*   The AWS IoT Device SDK for JavaScript requires Node v10.0 or later.
+```
+node -v
+```
 
-### Common Run Time
-The aws-crt package can be installed via npm 
+### Install the required libraries
+```
+sudo apt-get install cmake
+sudo apt-get install libssl-dev
+```
+
+### Install the common run time
```

Comment:
This step shouldn't be necessary, the CRT should come in as a transitive dependency.

Reply:
I would have thought so, but on some of my [many] Raspberry Pi installation attempts, I got errors that it couldn't be found. Adding these to the process made the errors go away. It seemed like something that would be [almost] a no-op if the libraries were present and would save the installation if they weren't. Note, that I didn't see any problems when installing the SDK on an Ubuntu EC2 instance. I only saw them on the Raspberry Pi with the May 2020 OS.